### ISSUE TITLE
Cursor Position

### ISSUE THREAD

**Issue by paragkan** [microsoft/vscode#105482]
If an open file in Visual Studio Code loses focus, the cursor position shifts to the beginning of the given line (if the current line is blank) instead of the current position. This happens only when the option "Formatting on Save" is selected in "Settings". It is irritating. Please resolve.

**Comment by RMacfarlane** [comment]
@paragkan Please include
- What version of VSCode you are using
- What extensions you have installed
- What types of files this happens for

**Comment by paragkan** [comment]
Hi Rachel : Thanks for your response.

   - What version of VSCode you are using








*Version: 1.48.2 (system setup)Commit:
a0479759d6e9ea56afa657e454193f72aef85bd0Date:
2020-08-25T10:13:11.295ZElectron: 7.3.2Chrome: 78.0.3904.130Node.js:
12.8.1V8: 7.8.279.23-electron.0OS: Windows_NT x64 10.0.19041*

   - What extensions you have installed

*I have many color theme extensions, I checked with the default color
theme, still the same issue.*

   - What types of files this happens for

*I have checked from three types of files:  .py .html .cpp *

*I did not observe that in css file. I believe that css file is not
auto-formatted upon "Save" or "Paste". *
